src/PIL/features.py

```diff
@@ -187,7 +264,15 @@ def pilinfo(out=None, supported_formats=True):
         ("xcb", "XCB (X protocol)"),
     ]:
         if check(name):
-            print("---", feature, "support ok", file=out)
+            if name == "jpg" and check_feature("libjpeg_turbo"):
+                v = "libjpeg-turbo " + version_feature("libjpeg_turbo")
+            else:
+                v = version(name)
+            if v is not None:
+                t = "compiled for" if name in ("pil", "jpg") else "loaded"
+                print("---", feature, "support ok,", t, "version", v, file=out)
```

Comment:
Perhaps this is a bit more readable? Should be obvious the number is a version.

```suggestion
                print("---", feature, "support ok,", t, v, file=out)
```

# Before

```
--------------------------------------------------------------------
--- PIL CORE support ok, compiled for version 7.2.0.dev0
--- TKINTER support ok
--- FREETYPE2 support ok, loaded version 2.10.1
--- LITTLECMS2 support ok, loaded version 20.90
--- WEBP support ok, loaded version 1.1.0
--- WEBP Transparency support ok
--- WEBPMUX support ok
--- WEBP Animation support ok
--- JPEG support ok, compiled for version 9.0
--- OPENJPEG (JPEG2000) support ok, loaded version 2.3.1
--- ZLIB (PNG/ZIP) support ok, loaded version 1.2.11
--- LIBTIFF support ok, loaded version 4.1.0
*** RAQM (Bidirectional Text) support not installed
*** LIBIMAGEQUANT (Quantization method) support not installed
*** XCB (X protocol) support not installed
--------------------------------------------------------------------
```

# After

```
--------------------------------------------------------------------
--- PIL CORE support ok, compiled for 7.2.0.dev0
--- TKINTER support ok
--- FREETYPE2 support ok, loaded 2.10.1
--- LITTLECMS2 support ok, loaded 20.90
--- WEBP support ok, loaded 1.1.0
--- WEBP Transparency support ok
--- WEBPMUX support ok
--- WEBP Animation support ok
--- JPEG support ok, compiled for 9.0
--- OPENJPEG (JPEG2000) support ok, loaded 2.3.1
--- ZLIB (PNG/ZIP) support ok, loaded 1.2.11
--- LIBTIFF support ok, loaded 4.1.0
*** RAQM (Bidirectional Text) support not installed
*** LIBIMAGEQUANT (Quantization method) support not installed
*** XCB (X protocol) support not installed
--------------------------------------------------------------------
```

Reply:
Good idea.